src/systemtest/resources/eiffel-intelligence.sh

```diff
@@ -0,0 +1,194 @@
+    call "mvn clean"
+    call "mvn package -DskipTests=true"
+    echo "Cloning Eiffel Intelligence back-end and building war file"
+    # Clone EI back-end and build war file
+    if [[ -d "eiffel-intelligence" ]]; then rm -Rf eiffel-intelligence; fi
+    call "git clone --depth=50 --branch=master https://github.com/eiffel-community/eiffel-intelligence.git"
+    cd eiffel-intelligence
+    call "mvn package -DskipTests=true"
+    cd ..
+}
+
+function do_start {
+    echo "Starting up Docker environment"
+
+    # Sets Docker image names and ports for containers.
+    # Also sets host variable
+    source src/main/docker/env.bash
+
+    echo "build: $build"
+    if [[ $build == "true" ]]; then
+        ## Set variables for Eiffel Intelligence war files.
+        export EI_FRONTEND_WAR_FILE=$(ls target/*.war)
+        export EI_BACKEND_WAR_FILE=$(cd eiffel-intelligence && ls target/*.war)
+
+        # Set up Docker containers and build the images of EI front-end, back-end and Jenkins
+        call "docker-compose -f src/main/docker/docker-compose.yml up -d --build"
+    else
+        # Pull images from DockerHub to avoid building images that doesn't exist
+        call "docker-compose -f src/main/docker/docker-compose.yml pull"
+        # Jenkins container always need to be built to install Groovy plugin properly
+        call "docker-compose -f src/main/docker/docker-compose.yml build jenkins"
+        # Start Docker containers
+        call "docker-compose -f src/main/docker/docker-compose.yml up -d"
+    fi
+
+    echo "Sleeping for 2 minutes, to let containers start up properly"
+    sleep 2m
+}
+
+function do_check {
+    echo "Checking if Jenkins container is up"
+    HTTP_CODE=503
+    until [[ $HTTP_CODE -eq "403" ]]; do
+        HTTP_CODE=$(curl -s -o /dev/null -w "%{http_code}\n" localhost:8082)
+        printf "."
+        sleep 5
+    done
+    printf " Jenkins container is up and running!\n"
+}
+
+function do_test {
+    echo "Starting system test"
+    # Set host variable
+    source src/main/docker/env.bash > /dev/null
+    verbose " Executing: mvn verify -P systemTest -Dei.frontend.url=http://${HOST}:8081 -Djenkins.external.url=http://${HOST}:8082 -B"
+    mvn verify -P systemTest -Dei.frontend.url="http://${HOST}:8081" -Djenkins.external.url="http://${HOST}:8082" -B
+    STATUS=$?
+}
+
+function do_stop {
+    echo "Stopping Docker containers ..."
+    # Sourcing environment variables to avoid warnings
+    source src/main/docker/env.bash > /dev/null
+    call "docker-compose -f src/main/docker/docker-compose.yml down"
+
+    # Cleaning up dangling docker images which were built using docker-compose command
+    verbose "Removing unused Docker images if there are any"
+    images=$(docker images -f "dangling=true" -q)
+    if [[ ! -z "$images" ]]; then call "docker rmi $images"; fi
+}
+
+function verbose {
+  if [[ "$verbose" -eq "1" ]]; then
+    echo -e $1
+  fi
+}
+
+function call {
+  verbose "Executing: '$@'"
+  $1
```

Comment:
what is this doing? Isn't $1 only one word/command from the whole command?
Shouldn't "call" be name exec or something? Since it looks like this function executes a cli command.

Reply:
Changed the name of the function